<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Connect BaseData</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_adressBase">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>info BD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tv_db"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lb_nameTable">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Введите имя таблицы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_setNameTable"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_dbShowRow">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Вывод строки по id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_entryId">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_dbShowAll">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Вывод всей базы и редактирование</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pb_dbConnect">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Соединение с базой данных</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_toolConnect">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Настройки соединения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>107</y>
      <width>184</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnectToBase"/>
    <addaction name="actionTool_connect"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Выход из программы</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnectToBase">
   <property name="text">
    <string>Connect DB</string>
   </property>
   <property name="toolTip">
    <string>Соединение с Базой Данных</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionTool_connect">
   <property name="text">
    <string>Tool connect</string>
   </property>
   <property name="toolTip">
    <string>Настройка соединения</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSet_Name_Table">
   <property name="text">
    <string>Set Name Table</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
